docker-compose.yml

```diff
@@ -49,7 +49,7 @@ services:
       context: .
       dockerfile: ./docker/idetect/Dockerfile
     image: idetect
-    command: sh -c 'python3 -m unittest /home/idetect/python/idetect/tests/test_*.py'
+    # command: sh -c 'python3 -m unittest /home/idetect/python/idetect/tests/test_*.py'
```

Comment:
Oh....I commented this out because I wanted to minimize resources while I was working on this, and I was monitoring the console and found all the testing output a bit distracting.